components/motor/gpio/motor_encoder.go

```diff
@@ -113,16 +88,17 @@ func newEncodedMotor(
 
 	cancelCtx, cancel := context.WithCancel(context.Background())
 	em := &EncodedMotor{
-		Named:            name.AsNamed(),
-		cfg:              motorConfig,
-		ticksPerRotation: int64(motorConfig.TicksPerRotation),
-		real:             localReal,
-		cancelCtx:        cancelCtx,
-		cancel:           cancel,
-		rampRate:         motorConfig.RampRate,
-		maxPowerPct:      motorConfig.MaxPowerPct,
-		logger:           logger,
-		opMgr:            operation.NewSingleOperationManager(),
+		Named:             name.AsNamed(),
+		cfg:               motorConfig,
+		ticksPerRotation:  int64(motorConfig.TicksPerRotation),
```

Comment:
can this be a float?

Reply:
yeah it can be. will still have to convert tho because the `motorConfig.TicksPerRotation` type is `int`

Reply:
that's okay - we can also change that when we change the config too. it's just that `ticksPerRotation` could be a float, and i think our encoder.Position API returns floats anyway.